assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
```

Comment:
It feels a little funny right if it returns `undefined` we cast that to a string and check for a hash?

IMO this feels more readable: 
```
focusId = $this.attr("href");
if (typeof focusId === 'undefined' || focusId.indexOf("#") !== 0)
 return;
}
```